Vagrantfile

```diff
@@ -67,7 +67,9 @@ sudo mkdir -p /etc/sysconfig
 sudo cp /home/vagrant/go/src/github.com/cilium/cilium/contrib/systemd/cilium-consul.service /lib/systemd/system
 sudo cp /home/vagrant/go/src/github.com/cilium/cilium/contrib/systemd/cilium-docker.service /lib/systemd/system
 sudo cp /home/vagrant/go/src/github.com/cilium/cilium/contrib/systemd/cilium-etcd.service /lib/systemd/system
+sudo cp /home/vagrant/go/src/github.com/cilium/cilium/contrib/systemd/cilium-health.service /lib/systemd/system
```

Comment:
I believe that in the previous commit the file didn't have `.service` in the filename

Reply:
I have a proof that it has :) https://github.com/cilium/cilium/pull/4596/files#diff-101273d12c622fcf417ca435ebcca97c There are two files, with and without `.service`. The file without is a file for `/etc/sysconfig`.